it/uk/gov/hmrc/enrolmentsorchestrator/controllers/AgentControllerISpec.scala

```diff
@@ -120,36 +122,39 @@ class AgentControllerISpec extends TestSetupHelper with LogCapturing {
 
   }
 
-  // DELETE /agent-client-enrolments/relationships/:arn/service/:service/client/:clientIdType/:clientId
   "DELETE /enrolments-orchestrator/relationships/:arn/service/:service/client/:clientIdType/:clientId" should {
```

Comment:
Pls change service name - a project check for "enrolments-orchestrator" should do it